app/controllers/publish_controller.rb

```diff
@@ -1,11 +1,10 @@
 class PublishController < FormController
-  before_action :assign_form_objects
+  before_action :assign_form_objects, :assign_autocomplete_warning
```

Comment:
is this because it is also required for the `create`?

Reply:
just a thought, but to save making the API call until only when it's necessary we could also do this just before we render `:index` on line 20.

Reply:
oh yeah this is a good suggestion. I don't have a strong preference out of the two options, happy with either